### PR TITLE
Add Lumi documentation

### PR DESCRIPTION
**Pull request by meircif**
Lumi documentation: http://lumi-lang.readthedocs.io/en/latest.
You are more than welcome to review the docs and write suggestions, comments and questions :smiley:.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 [![Build Status](https://travis-ci.org/meircif/lumi-lang.svg?branch=master)](https://travis-ci.org/meircif/lumi-lang)
 [![Build status](https://ci.appveyor.com/api/projects/status/wtam4i4gh28aw7su?svg=true)](https://ci.appveyor.com/project/meircif/lumi-lang)
 [![codecov](https://codecov.io/gh/meircif/lumi-lang/branch/master/graph/badge.svg)](https://codecov.io/gh/meircif/lumi-lang)
+[![Documentation Status](https://readthedocs.org/projects/lumi-lang/badge/?version=latest)](http://lumi-lang.readthedocs.io/en/latest/?badge=latest)
```
Comment (levioctl) [inline README.md]: Don't you want the link to point to the current version? (yes i realize it's more maintenance work)

File: docs/basics.rst [hunk 2]
```diff
@@ -0,0 +1,103 @@
+Basic Syntax
+============
+
+Comments
+--------
+In :ref:`TL4 <syntax-tl4>` Single line comments start with ``;``, multi-line
```
Comment (levioctl) [inline docs/basics.rst]: Don't you want to use a link directive that works both in readthedocs and in github's markdown (I haven't tested it but it looks like the links here: https://guides.github.com/features/mastering-markdown/ should work on both platforms).

File: docs/conf.py [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Lumi Language Guide'
+copyright = '2018, Lumi Language'
+author = 'Lumi Language'
```
Comment (levioctl) [inline docs/conf.py]: Author is 'Lumi Language'?

File: docs/index.rst [hunk 4]
```diff
@@ -0,0 +1,73 @@
+Lumi Language Guide
```
Comment (levioctl) [inline docs/index.rst]: I would consider steps for making the index page a bit more brief and welcoming.
For instance:
* Moving the 'hello world' link to an earlier point in the page
* Perhaps add a small 'Getting Started' section which is simply a few installation commands + hello world
* Moving the meaning of the name to a different page

File: docs/index.rst [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+The latest state of the language is named ":ref:`Temporary Lumi 4
+<syntax-tl4>`", or "TL4" in short, to emphasizing the current temporary state
+of the language.
+
+Contributing to Lumi
+--------------------
+Any help, suggestion, comment or questions is welcome! See Lumi repository
+`wiki`_ for more details.
+
+.. _wiki: https://github.com/meircif/lumi-lang/wiki
```
Comment (levioctl) [inline docs/index.rst]: Don't you want to use the docs instead of the wiki? (to have one 'source of truth')

Comment (meircif) [reply docs/index.rst]: What I panned is that:
* the docs explains the Lumi language
* the Wiki explains how to contribute to the lumi-lang repository

do you think it's better to have both in the docs?

File: docs/basics.rst (hunk 2 above)
Comment (meircif) [reply docs/basics.rst]: I will check this out...

**Pull request merged**